test/stages/stage1-gitops-bootstrap.tf

```diff
@@ -9,6 +9,4 @@ module "gitops-bootstrap" {
   sealed_secret_cert  = module.cert.cert
   sealed_secret_private_key = module.cert.private_key
   prefix              = var.bootstrap_prefix
-  kubeseal_namespace  = var.kubeseal_namespace
```

Comment:
Why did you remove `kubeseal_namespace` and `create_webhook`?   The namespace is passed in from environment variables b/c it can change per cluster, and the webook boolean creates a webhook, which allows the tests to execute much faster (b/c argo won't have to poll the git repo)

Reply:
Both of these should remain in the test case, and coudl be why you are getting that sealed secrets error.